.terraform.lock.hcl

```diff
@@ -19,22 +19,3 @@ provider "registry.terraform.io/hashicorp/aws" {
     "zh:fb338d5dfafab907b8608bd66cad8ca9ae4679f8c62c2435c2056a38b719baa2",
   ]
 }
-
-provider "registry.terraform.io/hashicorp/random" {
```

Comment:
It removed this when I ran `terraform init`.

Reply:
To generate lockfiles that work with all platforms, (linux and macos)

can you run this command and push a commit with the new lock file?

```
terraform providers lock \
  -platform=darwin_amd64 \
  -platform=linux_amd64
```

Also random being removed is probably fine, sounds like we aren't using it anymore in this module.